### PR TITLE
Docs - Changed docOps directory name to documentation

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* Changed docOps directory name to documentation
* Updated deploy-docs-to-gh-pages.yml workflow accordingly

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Documentation Fix

## How the change has been tested

Changes where tested locally.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: documentation/README.md [hunk 1]
```diff
@@ -23,11 +23,3 @@ yarn build
 ```
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
```
Comment (apenzk) [inline documentation/README.md:25]:
```suggestion
This command generates static content into the `build` directory and can be served using any static contents hosting service. E.g. in the browser attempt to load via http://localhost:3000/.
```

**Review by apenzk — APPROVED** [review]
Looks good

**Review by apenzk — CHANGES_REQUESTED** [review]
There should be an instruction in which file the sidebar in the docs is changed and organised. And how to do it

**Comment by lucas-tortora** [comment]
> Should there be an instruction about how to change the sidebar in the docs and how to reference files. I.e. discuss how the sidebars.js works. But where could this be put?

I've added new sections addressing these topics in the `documentation/README.md` file.

**Pull request closed without merging**